src/constraint_system/composer.rs

```diff
@@ -230,13 +232,36 @@ impl StandardComposer {
         // Append commitments to transcript
         ppc.seed_transcript(transcript);
 
-        ppc
+        Ok(ppc)
     }
 
     /// Returns the number of gates in the circuit
     pub fn circuit_size(&self) -> usize {
         self.n
     }
+
+    /// Checks that all of the wires of the composer have the same
+    /// length.
+    fn check_poly_same_len(&self) -> Result<(), PreProcessingError> {
```

Comment:
This function has been created, but is not called anywhere

Reply:
It is on preprocessing.

But merging your PR deleted a bunch of the stuff I had.